files/fr/learn/html/introduction_to_html/getting_started/index.md

```diff
@@ -463,7 +463,7 @@ Nous vous recommandons de toujours inclure les guillemets afin d'éviter ce type
 Dans cet article, vous remarquerez que les valeurs des attributs sont toutes entre des guillemets doubles ("&nbsp;"). Vous pouvez cependant voir des guillemets simples ('&nbsp;') dans le code HTML de certaines personnes. C'est purement une question de style, et vous êtes libre de choisir la solution que vous préférez. Les deux lignes suivantes sont équivalentes :
 
 ```html
```

Comment:
Could you tell me what I need to remove errors from my PR?

Reply:
`-nolint` suffix is used to prevent prettier from formating the code example. You just need to replace the language tag from `html` to `html-nolint`, and then the lint error would be resolved:

> click the "Commit suggestion" button, and commit the changes would take effect
>
> ![image](https://github.com/mdn/translated-content/assets/15844309/22831bf8-bc86-4942-9718-aa36be344022)

Reply:
Hi, @yin1999 
Thank you for your help.